Yesod/Auth/OAuth2.hs

```diff
@@ -17,12 +17,14 @@ module Yesod.Auth.OAuth2
 import Control.Exception.Lifted
 import Control.Monad.IO.Class
 import Data.ByteString (ByteString)
-import Data.Text (Text)
+import qualified Data.ByteString as BS
+import Data.Text (Text, append, pack)
```

Comment:
Would you mind importing `Data.Monoid.(<>)` and using that consistently in place of `Text.append`, `BS.append`?